go/src/client/main.go

```diff
@@ -260,8 +260,18 @@ func playMatch(baselinePath string, candidatePath string, params []string, flip
 		case best_move := <-p.BestMove:
 			err := game.MoveStr(best_move)
 			if err != nil {
-				log.Println("Error decoding: " + best_move + " for game:\n" + game.String())
-				return 0, "", "", err
+				log.Println("Assuming resign: " + best_move + " for game:\n" + game.String())
```

Comment:
I think rather than being in the error path this could be done something like...
if best_move == "(none)" {
  ... new code here.
} else {
  ... old code here.
}